apps/connect/source/ftrack_connect/ui/widget/asset_selector.py

```diff
@@ -31,11 +31,21 @@ def loadAssets(self, entity, selectAsset=None):
         '''
         assets = []
         try:
+            self.logger.debug('Entity type: {0}'.format(entity.entity_type))
             # ftrack does not support having Tasks as parent for Assets.
             # Therefore get parent shot/sequence etc.
             if entity.entity_type == 'Task':
+                self.logger.debug(
+                    'Entity is a Task, with id {0}, getting parent entity.'.format(
+                        entity['id']
+                    )
+                )
                 entity = entity['parent']
```

Comment:
As far as I remember, Tasks are not required to have a parent. This fails in that case.
(I might be wrong though)